docs/requirements.txt

```diff
@@ -0,0 +1,3 @@
+git+git://github.com/sphinx-doc/sphinx.git@8bf84167a30aa05886fcc1ed8895c8c20e939d89
```

Comment:
please update to versions used in this PR - https://github.com/luxonis/depthai-docs-website/pull/172